stdlib/LinearAlgebra/test/cholesky.jl

```diff
@@ -87,7 +86,6 @@ end
 
         apos = apd[1,1]
         @test all(x -> x ≈ √apos, cholesky(apos).factors)
-        @test_throws PosDefException cholesky(-one(eltya))
```

Comment:
(`cholesky` returns a failed decomposition rather than throwing a `PosDefException`, hence the test failure following translation of the call from `chol` to `cholesky`.)